db/osqlblockproc.c

```diff
@@ -1354,6 +1354,8 @@ static int process_this_session(
     uuid_t uuid;
 
     iq->queryid = osql_sess_queryid(sess);
+    iq->startprocessingus = comdb2_time_epochus();
+    reqlog_set_queue_time(iq->reqlogger, iq->startprocessingus - iq->startus);
```

Comment:
This will double count the queue time, and report higher latency for long requests.

Reply:
Yes, you are correct, all due to the nice math:
1770     logger->durationus =
1771         (comdb2_time_epochus() - logger->startus) + logger->queuetimeus;   
So I'm left with the dilemma of resetting logger->startus to current when assigning logger->queuetimeus (and making all this nastier).